wordpress/package.json

```diff
@@ -8,6 +8,7 @@
   "license": "MIT",
   "private": true,
   "scripts": {
+    "prestart": "sh ./scripts/pre-start.sh",
```

Comment:
@kuuak Is it a script that we have to launch ourself ? I think we should also add some documentation about this feature (+ this script if needed to run manually)

Reply:
@charl0tee The `prestart` script is run automatially every time `start` script is executed (just before as it's name suggests). See npm doc:

- [life-cycle-scripts](https://docs.npmjs.com/cli/v6/using-npm/scripts#life-cycle-scripts)
- [npm-start](https://docs.npmjs.com/cli/v6/using-npm/scripts#npm-start)

Reply:
@kuuak Ah ok ! Niice ! :👌